frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx

```diff
@@ -338,7 +362,76 @@ export const SessionRecordingPreview = memo(
                             )}
                         </div>
 
-                        <FirstURL startUrl={recording.start_url} />
+                        <div className="flex items-center justify-between gap-x-0.5">
```

Comment:
i'm torn on this one... every button we add is one more thing for folk to understand... I actually really like the idea but haven't heard users asking for it - is it you scratching your own itch or reacting to customer demand?

could we split this into two PRs?
one to add the 'apply filter from overview'
one to add the 'show more data in the playlist'

that way we can merge or not at a different time for each